ci/steps/build-docker-image.sh

```diff
@@ -1,12 +1,37 @@
 #!/usr/bin/env bash
```

Comment:
Should we rename this file? Maybe `build-and-push-docker-image.sh`?

Reply:
I did consider this but felt it was a bit of a mouthful - though am not opposed to it!

Reply:
It does seem a bit long haha but I don't want to mislead anyone who might try to run this!

Reply:
Could maybe shorten to `docker-buildx-push.sh` or something.

Reply:
Sounds great!